tests/TwilioFakeTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace Aloha\Twilio\Tests;
+
+use Aloha\Twilio\Support\Laravel\Facade as Twilio;
+use Orchestra\Testbench\TestCase;
+
+class TwilioFakeTest extends TestCase
+{
+    public function testItCanFakePhoneCalls()
+    {
+        Twilio::fake();
+
+        Twilio::from('default')->call('+15551234567', 'phone call fake');
+
+        Twilio::assertCallSent('+15551234567', 'phone call fake');
+        Twilio::assertCallSent('+15551234567', 'phone call fake', 'default');
+    }
+
+    public function testItCanFakeMessages()
+    {
+        Twilio::fake();
+
+        Twilio::message('+32474123456', 'message fake');
```

Comment:
Can I whatsapp you on this number, mister Belgium?

Reply:
If someone owns this faux *123456 number, they may be very confused about what a Laravel is.